FFMPEG_VERSION is undefined and actually not used elsewhere than in configure

--- transcode-1.0.2/configure.in~	2006-12-12 01:19:10.469599970 +0200
+++ transcode-1.0.2/configure.in	2006-12-12 01:25:43.288376547 +0200
@@ -626,17 +626,15 @@
     printf("install ffmpeg 0.4.9-pre1 or newer, or a cvs version after 20040703");
     return(1);
   }
-  printf("VER=%s\n", FFMPEG_VERSION);
   printf("BUILD=%d\n", LIBAVCODEC_BUILD);
   return(0);
 }
 ],
-    [LIBAVCODEC_VERSION="`./conftest$ac_exeext | sed -ne 's,VER=\(.*\),\1,p'`"
+    [
       LIBAVCODEC_BUILD="`./conftest$ac_exeext | sed -ne 's,BUILD=\(.*\),\1,p'`"],
     [TC_PKG_ERROR(libavcodec, ffmpeg/avcodec.h, required, FFmpeg, [http://www.ffmpeg.org],
       [cannot compile ffmpeg/avcodec.h])],
     [echo $ac_n "cross compiling; assumed OK... $ac_c"
-      LIBAVCODEC_VERSION=""
       LIBAVCODEC_BUILD=""])
   if test x"$enable_libavcodec_static" = x"yes" ; then
     if test x"$deplibs_check_method" != x"pass_all" ; then
@@ -657,7 +655,6 @@
   fi
   CPPFLAGS="$save_CPPFLAGS"
 fi
-AC_SUBST(LIBAVCODEC_VERSION)
 AC_SUBST(LIBAVCODEC_BUILD)
 AC_SUBST(LIBAVCODEC_CFLAGS)
 AC_SUBST(LIBAVCODEC_LIBS)
@@ -1433,7 +1430,6 @@
 headers                        $LIBAVCODEC_CFLAGS
 libraries                      $LIBAVCODEC_LIBS
 build                          $LIBAVCODEC_BUILD
-version                        $LIBAVCODEC_VERSION
 statically linked              $enable_libavcodec_static
 
 hardware support
